docs/api-reference/data-fetching/get-static-props.md

```diff
@@ -83,6 +83,8 @@ export async function getStaticProps() {
 
 Learn more about [Incremental Static Regeneration](/docs/basic-features/data-fetching/incremental-static-regeneration.md)
 
+The current cache status of a page leveraging ISR can be seen through the `x-nextjs-cache` header. When a path is first generated and not available in the cache the header will be `MISS`, when a path is in the cache but is passed the revalidate time it will be updated in the background and the header will be `STALE`, or when a path is in the cache and has not passed the revalidate time the header will be `HIT`.
```

Comment:
```suggestion
The cache status of a page leveraging ISR can be determined by reading the value of the `x-nextjs-cache` response header. The possible values are the following:

- `MISS` - the is not in the cache (occurs at most once, on the first visit)
- `STALE` - the path is in the cache but exceeded the revalidate time so it will be updated in the background
-  `HIT` -  the path is in the cache and has not exceeded the revalidate time
```